addons/account/models/sequence_mixin.py

```diff
@@ -268,7 +268,7 @@ def _is_end_of_seq_chain(self):
             seq = format_values.pop('seq')
             batch = batched[(format, frozendict(format_values))]
             batch['seq_list'].append(seq)
-            if batch['last_rec'].sequence_number < record.sequence_number:
+            if not batch['last_rec'] or batch['last_rec'].sequence_number < record.sequence_number:
```

Comment:
But you want to allow unlinking the move, don't you? You are still raising an error...
Doesn't this work?
```suggestion
            if batch['last_rec'].sequence_number <= record.sequence_number:
```

Reply:
I'm not sure. The invoice has sequence number equal to 0, which is not the last one. So it makes sense to block deleting, no?

Reply:
It is the last one, otherwise it would not be 0

Reply:
Fixed. It happened because `sequence_prefix` was computed as empty string. Please review again